provision/ubuntu/install.sh

```diff
@@ -71,8 +71,8 @@ cd ..
 rm -fr util-linux-2.30.1/ util-linux-2.30.1.tar.gz
 
 # Documentation dependencies
-pip3 install --user sphinx sphinxcontrib-httpdomain sphinxcontrib-openapi sphinx-rtd-theme sphinx-tabs recommonmark
-pip3 install --user yamllint
+sudo -H pip3 install sphinx sphinxcontrib-httpdomain sphinxcontrib-openapi sphinx-rtd-theme sphinx-tabs recommonmark
+sudo -H pip3 install yamllint
```

Comment:
Did you validate that this fix works locally? I'm worried we'll hit the same issues again in the CI (which the code that's removed below was supposed to fix :( ).

Reply:
yes, this is inslatedd on /usr/local/bin as now ;-)